### DISCUSSION TITLE
Full markdown file: all translations get reset whenever file changes

### DISCUSSION THREAD

**Discussion by mejo-** [WeblateOrg/weblate#6376] [Q&A]
Hello weblate community,

in our [Nextcloud Collectives app](https://gitlab.com/collectivecloud/collectives/) we have [an internationalized markdown file](https://gitlab.com/collectivecloud/collectives/-/blob/main/skeleton/Readme.en.md) and I added this as [a component](https://hosted.weblate.org/projects/collectives/nextcloud-collectives-app-landing-page-skeleton/) to be translated in Weblate.

Problem is that all translated strings get reset whenever the source file changes. As you can imagine, this is super annoying for translators.

So my questions is: do you know this problem and maybe even have a solution to it?

Kind regards
 Jonas

**Comment by nijel** [comment]
This is limitation of current implementation of some formats in Weblate (plain text, HTML, ODF and IDML).

**Reply by mejo-** [reply]
Thanks for the reply @nijel! So this basically means that that there's no easy solution to the problem?

Even more annoying is that weblate sometimes wants to overwrite translated parts of the file with the english string even though the translated strings are restored and accepted in weblate. See e.g. https://gitlab.com/collectivecloud/collectives/-/merge_requests/379/diffs

**Reply by nijel** [reply]
This looks like a bug - if the string is translated in Weblate, it should be translated in a file as well.

**Comment by mejo-** [comment]
@nijel is there any chance that this bug will be fixed in the near future? :blush:

**Reply by nijel** [reply]
You should start with creating an issue an describing what and when happens ;-)
